lib/rubocop/changes/check.rb

```diff
@@ -27,7 +27,7 @@ def line_numbers
       end
 
       def lines_from_diff
-        patch.changed_line_numbers
+        patch.nil? ? lines_from_rubocop : patch.changed_line_numbers
```

Comment:
this seems unrelated but `patch` was nil in the example I was checking 🤔

Reply:
We can review it together